<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Webtoon Formatter</string>
  </property>
  <widget class="QGroupBox" name="user_input_advance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>97</y>
     <width>511</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Advance Options</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="topMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QGridLayout" name="uia_layout" columnminimumwidth="0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="input_type_box">
        <property name="title">
         <string>Input Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>16</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="input_type_layout">
           <item>
            <widget class="QRadioButton" name="type_auto">
             <property name="text">
              <string>Auto</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="type_main">
             <property name="text">
              <string>Main</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="type_sub">
             <property name="text">
              <string>Chapter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="tolerance_box">
        <property name="title">
         <string>Tolerance [1-255]</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>32</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>32</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QSpinBox" name="tolerance_val">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QGroupBox" name="page_direction_box">
        <property name="title">
         <string>Page Direction</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>16</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="pd_layout">
           <item>
            <widget class="QRadioButton" name="page_direction_left">
             <property name="text">
              <string>Left &gt; Right</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="page_direction_right">
             <property name="text">
              <string>Right &gt; Left</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="min_panel_h_box">
        <property name="title">
         <string>Min Panel Height [%]</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>32</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>32</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="min_panel_h_val">
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QGroupBox" name="auto_output_box">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Output</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QCheckBox" name="auto_output_check">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Auto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="save_config_button">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>241</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="user_process">
    <item>
     <widget class="QLabel" name="status_box">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>READY</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="margin">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="process_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>60</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>FORMAT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>511</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="user_input_basic">
    <item row="1" column="0">
     <widget class="QLabel" name="out_dir_label">
      <property name="text">
       <string>Output Directory:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="in_dir_edit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Put Your Webtoon Directory Here</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="out_dir_edit">
      <property name="placeholderText">
       <string>Put Your Output Here</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="in_dir_button">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Paste</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="out_dir_button">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Paste</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="in_dir_label">
      <property name="text">
       <string>Input Directory:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
